nion/swift/Inspector.py

```diff
@@ -4994,3 +5015,22 @@ def data_item_delete() -> None:
             return widget
 
         return None
+
+
+class DeclarativeColorChooserConstructor:
+    def __init__(self, app: Application.Application) -> None:
+        self.__app = app
+
+    def construct(self, d_type: str, ui: UserInterface.UserInterface, window: typing.Optional[Window.Window], d: Declarative.UIDescription, handler: Declarative.HandlerLike, finishes: typing.List[typing.Callable[[], None]]) -> typing.Optional[UserInterface.Widget]:
+        if d_type == "color_chooser":
```

Comment:
Like the other PR, can you make this a qualified name like "nionswift.color_chooser" to make it easier to migrate this code to `nionui` in the future? Thanks!